### PR TITLE
Python Unittest Unhandled Exception

### PR DESCRIPTION
**Pull request by Dan-Nolan**
In unittest exceptions we'll show the exception class rather than the hardcoded exception string so  `<ERROR::>KeyError: 10` instead of `<ERROR::>Unhandled Exception: 10`

### CODE REVIEW

File: test/runners/python_spec.js [hunk 1]
```diff
@@ -133,6 +133,28 @@ describe('python runner', function() {
           done();
         });
       });
+      it('should show KeyErrors', function(done) {
+        runner.run({
+          language: 'python',
+          languageVersion: lv,
+          testFramework: 'unittest',
+          code: 'a = {}; a[10] += 3',
```
Comment (kazk) [inline test/runners/python_spec.js]: @Dan-Nolan these `testFramework` and `code` needs to be removed because ESLint is failing with "Duplicate key".

File: frameworks/python3/unittestwrapper.py [hunk 2]
```diff
@@ -27,9 +30,9 @@ def addSuccess(self, test):
     super(CwTestResult, self).addSuccess(test)
 
   def addFailure(self, test, err):
-    print("<FAILED::>" + formatMessage(err[1]))
+    print("\n<FAILED::>" + formatError(err))
     super(CwTestResult, self).addFailure(test, err)
 
   def addError(self, test, err):
-    print("<ERROR::>Unhandled Exception: " + formatMessage(err[1]))
+    print("\n<ERROR::>" + formatError(err))
```
Comment (kazk) [inline frameworks/python3/unittestwrapper.py]: I don't think this file (or any file in frameworks/python3) is used?

https://github.com/Codewars/codewars-runner-cli/blob/dbf943d62c8d6ed451d27ac1817c0f13b1de558c/lib/runners/python.js#L110

No need to revert the change though.

Comment (Dan-Nolan) [reply frameworks/python3/unittestwrapper.py]: Yeah I didn't think so either, fixed it just in case it gets used again 😄

File: test/runners/python_spec.js (hunk 1 above)
Comment (Dan-Nolan) [reply test/runners/python_spec.js]: Ah thanks! I'll fix this up.

**Comment by jhoffner** [comment]
@Dan-Nolan I guess I broke something when I try to merge the conflict.

**Comment by kazk** [comment]
This is failing because unittest with Python 3 now uses the no-concatenation introduced in #552, so the error output looks like the following:

```
<ERROR::>Unhandled Exception
<LOG:ESC:Error>KeyError: 10
<LOG:ESC:Traceback>  File "/home/codewarrior/test/test_solution.py", line 5, in test_assert<:LF:>    self.assertEqual(go(1), 1)<:LF:>  File "/home/codewarrior/solution.py", line 3, in go<:LF:>    a[10] += 3
```

and doesn't contain `<ERROR::>KeyError: 10`

**Comment by kazk** [comment]
Closing because this is no longer used.
